src/process.cr

```diff
@@ -139,6 +140,14 @@ class Process
   # will be closed automatically at the end of the block.
   #
   # Returns the block's value.
+  #```  
+  # someId = "123ABC456DEF"
```

Comment:
most of the Crystal examples use snake case.  You might want to change this to be similar to other examples.

```suggestion
  # some_id = "123ABC456DEF"
```